### DISCUSSION TITLE
Selected e_coefficient

### DISCUSSION THREAD

**Discussion by tjelse** [firemodels/fds#13811] [General]
Hello, I am working on a model to see the effects of a sprinkler system on the HRR of a tracker trailer heavy goods vehicle in a road tunnel. I see that  'e_coefficient' is only mentioned once in the FDS user manual and is stated to derived experimentally. This being said, I still need to put a value in to see the model results in this sprinkler model. Does anyone know the range of possible values, if there is a range? The model referenced in the FDS users manual uses a value of 1. If the fire needs this e_coefficient, does that mean any sprinkler model in FDS that wasn't experimentally measured for is inaccurate? Thanks for any and all help.

**Comment by mcgratta** [comment]
There is no list of these values. The only way to obtain a value is to do a large scale experiment with the exact type of cargo and configuration. As you can imagine, there are an infinite number of possibilities, which is why there is no list.

**Reply by tjelse** [reply]
That makes sense. Is it right to say without an accurate e_coefficient value, sprinkler models in FDS aren't entirely accurate for HRR reduction? I'm wondering how to go about a sprinkler modeler if there is no experimental data for e_coefficient.

**Reply by mcgratta** [reply]
Most use FDS to estimate the time of first sprinkler activation. Assumptions are made after that about the HRR. For example, some hold the HRR steady, but even that has to be justified based on the sprinkler system and commodity type.

**Reply by tjelse** [reply]
I see. Thank you for such prompt responses.
